parser/prism/Translator.cc

```diff
@@ -645,6 +645,16 @@ unique_ptr<parser::Node> Translator::translate(pm_node_t *node) {
             // Will only work for positive, 32-bit integers
             return make_unique<parser::Integer>(location, std::to_string(intNode->value.value));
         }
+        case PM_INTERPOLATED_MATCH_LAST_LINE_NODE: { // A test of the last read line with interpolation,
```

Comment:
`A test`? I don't quite get what this means.

Reply:
Thanks for the context, and WOW.
Maybe something like: `An interpolated regex in a conditional that checks last read line from IO`?

Reply:
I like Stan's description!

Reply:
I had to break it on 2 line to pass the formatter. That gave me some extra room, so I added some more detail:

```suggestion
        case PM_INTERPOLATED_MATCH_LAST_LINE_NODE: { // An interpolated regex literal in a conditional...
            // ...that implicitly checks against the last read line by an IO object, e.g. `if /wat #{123}/`
```